### DISCUSSION TITLE
Documentation suggestion to avoid risky rulesets confusion

### DISCUSSION THREAD

**Discussion by uuf6429** [PHP-CS-Fixer/PHP-CS-Fixer#8022] [Q&A]
I was about to create a bug report for the following config because it didn't seem to work / do anything:
```php
<?php

return (new PhpCsFixer\Config())
    ->setRiskyAllowed(true)
    ->setRules(['@PER-CS2.0:risky' => true])
    ->setFinder(
        (new PhpCsFixer\Finder())
            ->in(__DIR__)
    );
```
Then I realised that the problem is that `@PER-CS2.0:risky` does not contain `@PER-CS2.0` rules - so simple things like old array syntax were ignored.

I get why having rule sets like that might make sense. A closer inspection of https://cs.symfony.com/doc/ruleSets/PER-CS2.0Risky.html compared to https://cs.symfony.com/doc/ruleSets/PER-CS2.0.html (esp. the "Rules" section) also gives an indication of the difference.

I still have a small suggestion that might avoid confusing people. In case of the [risky rule set](https://cs.symfony.com/doc/ruleSets/PER-CS2.0Risky.html), keeping in mind that people will probably read the summary/description first, I suggest the following wording (at the moment it's a verbatim copy of the non-risky one):
```diff
- Rules that follow [PER Coding Style 2.0](https://www.php-fig.org/per/coding-style/).
+ Risky rules that follow [PER Coding Style 2.0](https://www.php-fig.org/per/coding-style/).
```
(so that readers know that it's specifically about risky rules and not all rules related to the standard)

Of course it probably makes sense to apply the same thing to all the other (risky) rulesets.

**Comment by Wirone** [comment]
Yes, I agree it would be more appropriate 🙂. Also, @keradus is there any reason why risky sets don't extend regular sets? It would actually make sense if `@PER-CS2.0:risky` would contain all the rules from `@PER-CS2.0` + risky rules.

**Comment by julienfalque** [comment]
Since we have a parameter to enable/disable usage of risky rules, does it make sense to have separate rulesets anyway?

**Reply by Wirone** [reply]
That's a good question indeed 😅. Maybe risky rules should be part of the regular sets, but should be skipped if `ConfigInterface::getRiskyAllowed()` returns `false`?
